test/test_enqueue_token.rb

```diff
@@ -0,0 +1,66 @@
+require 'minitest/autorun'
+require 'queue_it_token'
+
+class EnqueueTokenTest < Minitest::Test
+  def test_minimum_header
```

Comment:
You can add a `def setup` with @token_identifier and other shared values in test

Reply:
Done 👍